src/main/java/uk/gov/companieshouse/api/accounts/model/ixbrl/Account.java

```diff
@@ -1,14 +1,18 @@
 package uk.gov.companieshouse.api.accounts.model.ixbrl;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import uk.gov.companieshouse.api.accounts.model.ixbrl.balancesheet.BalanceSheet;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.company.Company;
 import uk.gov.companieshouse.api.accounts.model.ixbrl.notes.Notes;
 import uk.gov.companieshouse.api.accounts.model.ixbrl.period.Period;
 
 public class Account {
 
     private Period period;
```

Comment:
Could we add the `@JsonProperty` to the other fields as well, this is just to safeguard us against problems in the future if someone was to change the name of the fields as by default they are named after the variable itself. So for example if someone was to change `period` to `periodNew` it would break. We had this problem when renaming the models for the API.

Reply:
good point. I had thought the same but I didnt changed ;)